hoomd/mpcd/Integrator.h

```diff
@@ -67,6 +68,8 @@ class PYBIND11_EXPORT Integrator : public hoomd::md::IntegratorTwoStep
         syncCellList();
         }
 
+    void setRigid(std::shared_ptr<hoomd::md::ForceComposite> new_rigid) override;
```

Comment:
I don't think you need the fully qualified namespace. `ForceComposite` on its own will probably work fine, or you could use `md::ForceComposite` if you want to make clear which it comes from.
```suggestion
    void setRigid(std::shared_ptr<ForceComposite> new_rigid) override;
```

Reply:
The problem is that if I do that, then when I try to build with ninja it can't figure out what the ForceComposite is.